ui/app/components/config-pki-ca.js

```diff
@@ -52,7 +52,7 @@ export default Component.extend({
    * @param DS.Model
    * @public
    *
-   * a `pki-config` model - passed in in the component useage
```

Comment:
I also found this spelling in the config-pki.js row 23. Nice clean up work. Thank you.